client/my-sites/stats/stats-page-view-tracker/use-tracks.tsx

```diff
@@ -0,0 +1,52 @@
+import config from '@automattic/calypso-config';
+import debugFactory from 'debug';
+import { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { recordTracksEvent, enhanceWithSiteType } from 'calypso/state/analytics/actions';
+import { withEnhancers } from 'calypso/state/utils';
+import type { StatsPageViewTrackerProps } from './index';
+import type { AnyAction, Store } from 'redux';
+import type { ThunkDispatch } from 'redux-thunk';
+
+const debug = debugFactory( 'calypso:my-sites:stats:StatsPageViewTracker:useTracks' );
+
+const CALYPSO_STATS_PAGE_VIEW_EVENT_NAME = 'calypso_stats_page_view';
+const ODYSSEY_STATS_PAGE_VIEW_EVENT_NAME = 'odyssey_stats_page_view';
```

Comment:
I should have thought of this earlier, but it didn't occur to me until I saw the other comments - how about we use `jetpack_odyssey_stats_page_view` instead. I don't think we want to add a whole new `source` for Tracks, unless we are planning on adding a bunch more `odyssey` Tracks events.

Reply:
All good. I waited for your review because I thought there might be something like this we missed 🙂